apps/wallet-mobile/src/features/ListTxHistory/useCases/ListTxHistory/TxHistory.tsx

```diff
@@ -44,6 +44,8 @@ export const TxHistory = () => {
       end={{x: isDark ? 0 : 0, y: isDark ? 0.5 : 0}}
       style={styles.root}
     >
+      {Platform.OS === 'android' && <StatusBar translucent backgroundColor="transparent" />}
```

Comment:
@banklesss can it be done in the `useStatusBar` ?

Reply:
let me check it 👍🏻

Reply:
done